### PR TITLE
Kebbin proofreading

### PR DESCRIPTION
**Pull request by Condo-Colin**
Hello,
What is the standard reference to this system you've built?
The logo looks good with the "++", and that conveys the connection with Gregg's original.
When written, I see both "-EX" and "++ EX" with a space.
I like "DCC-EX" as a simple name to cover the suite of products; easy to say, easy to write, and ties in with the component names.
Maybe on the intro webpage we need to say something like "DCC++ EX, also referred to as DCC-EX." ("AKA" looks a bit acronym-heavy in that sentence!)
What do you think?

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ Welcome! DCC++ EX is an easy to use, do-it yourself and affordable, open source
 Mission
 #######
 
-Our mission is to make the world of model railroading universally accessible.
+Our mission is to make the world of digital model railroading universally accessible.
```
Comment (Condo-Colin) [inline docs/index.rst:40]: I didn't want to just change your Mission Statement!
This is just a suggestion, that what you're working on is for digital model railroading specifically.
What do you think?

**Comment by FrightRisk** [comment]
Remember to "pull" from master before creating your branch to merge in the latest changes.

I'm not sure what you mean by "what's the standard reference" to what I built. Elaborate. DCC-EX is the company, DCC++EX is the product. DCC-EX also makes WebThrottle-EX. But we have conflated the company name with the CS. I've been hoping its not confusing since the more we push people to DCC-EX over time, the less we will need the connection to the old DCC++.

I'm not sure about the distinction between digital model railroading and just model railroading. Does the word digital buy us anything? Is it cool sounding? We are trying to make ALL model railroading easier for people since it is a convoluted mess. And DCC has all but replaced DC. So does it go without saying that everything these days is digital? Or is it important to make the distinction because DC is still out there? Is anybody going to know or care? Back at you.

**Comment by Condo-Colin** [comment]
Hey Fred, thanks for the info!

> Remember to "pull" from master before creating your branch to merge in the latest changes.

I'm just using the GitHub online website tool, and I looked but couldn't find any "pull" or "refresh" option for my branch.
I then assumed after you did the merge the other day, and it seemed to rebranch mine out again, that mine would be up-to-date?
I just renamed mine last night for clarity.

> I'm not sure what you mean by "what's the standard reference" to what I built. Elaborate. DCC-EX is the company, DCC++EX is the product. DCC-EX also makes WebThrottle-EX.

DCC-EX, DCC++ EX, and now DCC++EX (no space) have all been used to name the developer's group, the product suite, and the Command Station. So there's also a company involved?
In most cases it is really inconsequential what name is used, (let's just run some trains! 🤩) but I was just trying to get some consistency across the webpages to start with.

> But we have conflated the company name with the CS. I've been hoping its not confusing since the more we push people to DCC-EX over time, the less we will need the connection to the old DCC++.
 
No, this is all good. It's like saying "I use Lenz, or NCE, or Whatever."

> I'm not sure about the distinction between digital model railroading and just model railroading. Does the word digital buy us anything? Is it cool sounding? We are trying to make ALL model railroading easier for people since it is a convoluted mess. And DCC has all but replaced DC. So does it go without saying that everything these days is digital? Or is it important to make the distinction because DC is still out there? Is anybody going to know or care? Back at you.

DC sure is still out here. Well, down here in the dark underbelly of the world as you all know it anyway! I've got maybe nearly a dozen DC locos, and quite a few basic DC controllers, and one solitary DCC sound decoder waiting to get installed. I was hoping to make some of my own decoders from Geoff Bunza's ideas one day.
Everything is going DCC, but my feeling is it's really only the well-off that have that luxury yet, but there is a lot of wealth in our lucky country I must say.
I guess DCC-EX could certainly help DC modellers automate their layouts without actually "commanding" the locos.
Maybe in your roadmap you might have wanted to include some DC stuff, but I think if we get some basic decoders happening then DC can be out the window, the window, the second story window. With a heave and a ho, and a great big throw, we'll throw it out the window!

Cheers, Colin.

File: docs/get-started/installer.rst [hunk 2]
```diff
@@ -7,9 +7,9 @@ Requirements
 * Windows/Linux/MacOS Computer
 * DCC-EX Compatible motor shield / motor controller board
 * DCC-EX Compatible Microcontroller (Arduino Mega/Uno)
-* USB cable to connect your computer to the Microcontroller Trains
```
Comment (Condo-Colin) [inline docs/get-started/installer.rst:10]: Hello,
The USB cable doesn't connect to the trains, but did this mean to say something else?
Like Train Controller maybe? Or was left over from an earlier wording?
Cheers, Colin.

Comment (FrightRisk) [reply docs/get-started/installer.rst:10]: LOL. So glad someone is going through all this word by word.

**Review by FrightRisk — APPROVED** [review]
Good to go. I'll correct the USB cable line

**Review by FrightRisk — APPROVED** [review]

**Pull request merged**